src/main/java/uk/gov/hmcts/reform/prl/clients/ccd/CaseAssignmentService.java

```diff
@@ -164,6 +165,17 @@ public void removeBarrister(final CaseData caseData, String selectedPartyId) {
         selectedParty.setBarrister(null);
     }
 
+    public void removeAmBarristerCaseRole(final CaseData caseData,
+                                Map<Optional<SolicitorRole>, Element<PartyDetails>> selectedPartyDetailsMap) {
+        if (featureToggleService.isAddBarristerIsEnabled()) {
```

Comment:
The feature flag name shows add barrister instead of remove barrister

Reply:
My understanding was flag was for an entire barrister feature.